<ui version="4.0" >
 <class>ThresholdLineEnhancerClass</class>
 <widget class="QWidget" name="ThresholdLineEnhancerClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ThresholdLineEnhancer</string>
  </property>
  <widget class="QFrame" name="frame_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>60</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QSpinBox" name="EditSelfSearch" >
    <property name="geometry" >
     <rect>
      <x>320</x>
      <y>40</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_30" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text" >
     <string>強調係数</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxEmphasizeRate" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>10</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals" >
     <number>4</number>
    </property>
    <property name="minimum" >
     <double>-99.989999999999995</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_33" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text" >
     <string>線幅</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditMaxLineWidth" >
    <property name="geometry" >
     <rect>
      <x>130</x>
      <y>40</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum" >
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_38" >
    <property name="geometry" >
     <rect>
      <x>210</x>
      <y>40</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape" >
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow" >
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text" >
     <string>探索距離</string>
    </property>
    <property name="alignment" >
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="ButtonReflectAllBlocks" >
   <property name="geometry" >
    <rect>
     <x>220</x>
     <y>140</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>同一種類のブロックに反映</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonClose" >
   <property name="geometry" >
    <rect>
     <x>220</x>
     <y>220</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>閉じる</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonSaveToLibrary" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>180</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>ライブラリに保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonLoadFromLibrary" >
   <property name="geometry" >
    <rect>
     <x>220</x>
     <y>180</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>ライブラリから読み出し</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidgetParts" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>140</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentIndex" >
    <number>0</number>
   </property>
   <widget class="QWidget" name="page" >
    <widget class="QPushButton" name="ButtonReflectOnlyBlock" >
     <property name="geometry" >
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>このブロックだけに反映</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3" >
    <widget class="QPushButton" name="ButtonReflectSaveParts" >
     <property name="geometry" >
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text" >
      <string>部品保存</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2" />
  </widget>
  <widget class="QPushButton" name="pushButtonChangeLib" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>30</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditLibID" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>30</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>12345</string>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text" >
    <string>ライブラリＩＤ</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text" >
    <string>ライブラリ名称</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditLibName" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>30</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>XXX</string>
   </property>
   <property name="readOnly" >
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
